week-2-practise/level-3/6-map.js

```diff
@@ -11,11 +11,13 @@ function tidyUpString(str) {
 }
 
 function captialise(str) {
-  // complete this function
+  str = str.split("");
+  str[0] = str[0].toUpperCase();
```

Comment:
Hey! This is how I wish I'd done the splitting! Very clean , very quick, very simple. 

Were you able to get the code to execute both the tidyUpString and capitalise function? I can't see tidyUp being called?